composer.json

```diff
@@ -31,7 +31,8 @@
         "symfony/http-foundation": "^4.4 || ^5.0",
         "symfony/phpunit-bridge": "^5.3",
         "symfony/routing": "^4.4 || ^5.0",
-        "twig/twig": "^1.40 || ^2.9 || ^3.0"
+        "twig/twig": "^1.40 || ^2.9 || ^3.0",
+        "phpspec/prophecy-phpunit": "^2.0"
```

Comment:
this is in the wrong position

Reply:
it is under require-dev. where should be placed phpspec/prophecy-phpunit?

Reply:
it begins with a "p" and it's last... can you guess where it should be placed?

Reply:
Done in https://github.com/KnpLabs/KnpMenu/pull/352/commits/db5d9a54bfa5d8da2bbb5926e48c1f7360ff9f60